### ISSUE TITLE
Native windows build

### ISSUE THREAD

**Issue by agrandville** [benmcollins/libjwt#190]
Ben,
would you accept a rework of the CMake build scripts for windows ?
I suggest using vcpkg to manage dependencies and appveyor for CI and unit tests ?

**Comment by benmcollins** [comment]
Absolutely. I don't personally do any Windows builds, nor do I use the CMake setup. I consider those "contrib" from helpful people like yourself. As long as it doesn't complicate or break the Auto Tools build, I'll accept it. Thanks!

**Comment by benmcollins** [comment]
Can this be closed?

**Comment by benmcollins** [comment]
I've switched over completely to CMake for the build. Please (anyone) provide patches for building on Windows.

**Comment by benmcollins** [comment]
Closing for now.
